src/components/common/MenuTab.tsx

```
@@ -1,34 +1,56 @@
     const sizeDesign = {
         "button": {
             "base": "text-base",
             "lg": "text-lg"
         }
     }
-    return <nav className={clsx("flex space-x-8", className)}>
-        {tabs.map((tab) => (
+    const tabRefs = useRef<(HTMLButtonElement | null)[]>([])
+    const [underlineStyle, setUnderlineStyle] = useState({ left: 0, width: 0 })
+
+    useLayoutEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: tabs.indexOf

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)